Update examples before upmerge (#4)
ArekBalysNordic/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

* Added:

  * Release configuration for all samples.
  * A binding cluster to the Light Switch sample.
  * The Groupcast communication to the Light Switch example.

* Updated:

>   * Regenerated all ZAP configurations

Reviewer: ```suggestion
  * All ZAP configurations.
```
"Updated: ... All ZAP configurations."

Can you mention why/what was updated?

>   * Changed a Pairing process to Binding process in Light Switch sample

Reviewer: ```suggestion
  * Pairing process to Binding process in the Light Switch sample.
```
"Updated: ... Pairing process to (...)"
